.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: ruby
-script: "rake travis:ci"
+script: "bundle exec rake travis:ci"
 rvm:
-  - 2.0.0
```

Comment:
Why did you drop support for older versions of Ruby?  This is as far as I can tell unrelated to this PR?